tests/eBPF/network_viewer.conf

```diff
@@ -0,0 +1,31 @@
+# This file requires a recent version of network viewer
+#
+# and the eBPF program installed with network-installer.sh case you do not have it, you can do download running:
+#
+
+# Outbound Range
+# 
+# Specify a valid ip range to monitor.
+#
+outbound = 10.0.0.0/8 172.16.0.0-255.240.0.0 192.168.0.0/255.255.0.0
+
+# Inbound Range
+# 
+# Specify a valid ip range to monitor.
+#
+inbound = 0.0.0.0/0
+
+#
+# Ports
+#
+# Specify a valid number of ports to monitor, please, pay attention for the fact that only 50 ports will be monitored.
```

Comment:
Why 50 is the cap and how to change it?

Reply:
We talked this during a grooming session, I think was @vlvkobal suggestion , should we allow more? This has a direct impact in the chart rendering.

Reply:
i think user configurable is ok

Reply:
Probably cap is not needed at all

Reply:
I will do a test without restrictions to check what can happen, I will also add a new parameter for the people to specify the maximum number of ports they want to see.

Reply:
My point that it is up to a user.
It will be very frustrating to tune that max thing again and again.

dsts: 1,2,3,4,5,6,7,8,9.................4545

I dont want to count number of ports as a user, i want them to work. 
And i need to change that `max` thing every time when i add another port in the line.

Just add a note there:  > N may affect performance. Please dont add `max`.

Reply:
You convinced me, I changed the documentation giving an advice and we are starting with only 50, but the user is free to set the ports that he wants.
I will work to bring port range for the user  do not need to list a big range of ports in their configuration file.